src/vireshJJun20/assignment36/LinkedHashSetEx1.java

```diff
@@ -0,0 +1,46 @@
+/*Assignment 36 : 31st July 2020
+
+Remove duplicate names from given String Array, print names in given format with its length. Input Order has to be maintained.
+String input[] = {"Aashtha","Kartikey","Piyush","Raj","Suparna","Piyush","Aashtha", "Barkha"}
+
+Output : 	Aashtha -> 7
+            Kartikey -> 8
+            Piyush -> 6
+	      	Raj -> 3
+	      	Suparna -> 7
+	      	Barkha -> 6
+Total unique elements are :- 6
+
+Note: Use set interface. */
+
+package vireshJJun20.assignment36;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class LinkedHashSetEx1 {
+
+	static Set<String> convertIntoLinkedHashSet(String input[]) {
+		System.out.println("Input Array : " + Arrays.toString(input));
+		Set<String> tempSet = new LinkedHashSet<String>(input.length);
+		for (int index = 0; index < input.length; index++) {
+			tempSet.add(input[index]);
```

Comment:
we can directly write below , no need to add with add method.
Set<String> s1 = new LinkedHashSet<String>(Arrays.asList(input));